keyboards/xd75/keymaps/scheiklb/config.h

```diff
@@ -0,0 +1,116 @@
+/* Copyright 2018 Kolja Brauns
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+// place overrides here
+
+// normal characters
```

Comment:
These should not be needed. 

instead you should use `#include "keymap_german.h'`, instead.  That file contains all of these defines already: 
https://github.com/qmk/qmk_firmware/blob/master/quantum/keymap_extras/keymap_german.h